src/main/java/com/powsybl/openloadflow/OpenLoadFlowParameters.java

```diff
@@ -1934,11 +1936,15 @@ public static DcLoadFlowParameters createDcParameters(LoadFlowParameters paramet
         return new DcLoadFlowParameters()
                 .setNetworkParameters(networkParameters)
                 .setEquationSystemCreationParameters(equationSystemCreationParameters)
+                .setSlackDistributionFailureBehavior(parametersExt.getSlackDistributionFailureBehavior())
                 .setMatrixFactory(matrixFactory)
                 .setDistributedSlack(parameters.isDistributedSlack())
+                .setAreaInterchangeControl(parametersExt.isAreaInterchangeControl())
                 .setBalanceType(parameters.getBalanceType())
                 .setSetVToNan(true)
-                .setMaxOuterLoopIterations(parametersExt.getMaxOuterLoopIterations());
+                .setMaxOuterLoopIterations(parametersExt.getMaxOuterLoopIterations())
+                .setSlackBusPMaxMismatch(parametersExt.getSlackBusPMaxMismatch())
```

Comment:
A naive question. Is the mismatch really needed given that the equations are linear ? Don't you just need an epsilon suitable for double comparison instead ?

Reply:
I think that there can be some (really) corner cases (with tielines not considered for interchange and a sum of interchange targets not exactly equal to 0), where using the slackBusPMax will make the outer loop tolerate a situation that would have failed if using a small epxilon.
I admit that I never created such test case... whether this is truly possible remains to be proven

Reply:
In the outerloop we have two phases : 

1) Check for each area if the interchange is closer to the target than AreaInterchangePmaxMismatch, and distribute the difference if not

When all areas are at their target : 

2) Check if some mismatch remains on the slack bus (with SlackBusPMaxMismatch) and distribute the difference : 
    - to the buses without Area first
    - to the buses with area if some mismatch still remains
   
There can be mismatch at stage 2) if the sum of the targets is not exactly 0 or if we have interconnections not accounted in the interchange so there is a difference between areas target and actual interchanges.
This situations make that some P mismatch will be present on the slack bus. 

Let's take the example where we have a network with all buses having an Area and where this slack mismatch is under SlackBusPMaxMismatch. 

- In case we use SlackBusPMaxMismatch as a threshold, this mismatch is tolerated and the LF is sucessful.

- In case we use a small epsilon, two scenarios : 
   - (most likely) : the mismatch is distributed to all network, and all Areas stay close enough to their interchange targets -> LF success and we just made 1 more iteration of outer loop
   - (ultra corner case imo) : the mismatch is distributed but some Area's interchange was very close to [target + AreaInterchangePmaxMismatch] (possible in DC if unconsidered interconnections) and this distribution of slack makes the Area be not balanced anymore -> adds at least 2 iterations to the OL or even more because the scenario could happen again.
   
To summarize, the important point is that the possibility of unconsidered interconnections adds uncertainty on the amount of power distributed by an Area, that can be different from its P imbalance.
In that context, I feel more confident to keep some tolerance on the slack mismatch threshold. 
But I admit that the probability of having serious issues due to this with that outerloop is very small, and would be ready to use an epsilon instead if you prefer, because indeed the benefits seem to outweigth the risks.